Makefile

```diff
@@ -105,3 +107,17 @@ release: verify release-src release-bin
 	shasum -a 512 $(RELEASE_SRC).tgz > $(RELEASE_SRC).tgz.sha512
 	gpg --batch --yes --armor --detach-sig $(RELEASE_BIN).tgz
 	shasum -a 512 $(RELEASE_BIN).tgz > $(RELEASE_BIN).tgz.sha512
+
+.PHONY: docs-gen
+docs-gen:
+	-if [ ! -f "plantuml.jar" ]; then curl -sL -o plantuml.jar https://repo1.maven.org/maven2/net/sourceforge/plantuml/plantuml/$(PLANTUML_VERSION)/plantuml-$(PLANTUML_VERSION).jar; fi;
+	-java -jar plantuml.jar -tsvg -nometadata "docs/*.plantuml"
```

Comment:
I think we can put `plantuml.jar` into `.gitignore` or place this file elsewhere (like a temporary folder) so that we won't check it into the codebase by accident?

Reply:
@kezhenxu94 You're right, it's already part of the `.gitignore`

Reply:
Oops, sorry for missing that